src/main/java/com/google/api/codegen/transformer/PackageMetadataTransformer.java

```diff
@@ -27,6 +27,14 @@
       String template,
       String outputPath,
       TargetLanguage language) {
+    // Note that internally, this is overridable using a "discovery" section in the service config,
+    // which is not available externally. See:
+    //   https://github.com/googleapis/toolkit/issues/933
+    String discoveryApiName = model.getServiceConfig().getName();
```

Comment:
The name `apiName` is confusing in the context of GAPIC given that we already have `shortName` and `fullName` to refer to the API's name. I want to clarify that this value should be used where the name of the discovery doc is needed.